src/LocaleStrings.lua

```diff
@@ -403,7 +403,7 @@ LC_RU.menuSkills = "Проверить доступность навыков"
 LC_EN.acquired = "You have acquired the (.*) skill%."
 LC_DE.acquired = "Ihr habt Euch die Fertigkeit (.*) angeeignet%."
 LC_FR.acquired = "Vous avez acquis la compétence (.*)%."
-LC_RU.acquired = "You have acquired the (.*) skill%."
+LC_RU.acquired = "Вы приобрели навык (.*)%."
```

Comment:
Has this text been confirmed and/or tested in an RU client by performing a skill acquisition and observing the skill getting added to the main window?

Reply:
Have not been able to test this. @Svertalf, can you help us understand how you achieved this?

Reply:
@Svertalf if you are not aware, this has to be the exact string as posted in chat by the game. I tested with the RU client using the downloaded DAT files straight from discord, and it is different from what you posted.

Reply:
I tested PR #215 in the RU client (downloaded dat files from discord).

Reply:
FInal version is ```LC_RU.acquired = "Вы приобрели умение <(.*)%>."```. This way skill acquisition detection in RU will start working.